tests/test_map.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+"""
+Folium map Tests
+----------------
+
+"""
+from folium.map import Popup  # TODO: Map, Marker, ...
+
+
+tmpl = """
+        <div id="{id}"
+                style="width: {width}; height: {height};">
+                {text}</div>
+""".format
+
+
+def test_popup_ascii():
+    popup = Popup('Some text.')
+    _id = list(popup.html._children.items())[0][0]
```

Comment:
I would have done

    _id = list(popup.html._children.keys())[0]

but it's really equivalent.